sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFileFormat.scala

```diff
@@ -160,6 +160,9 @@ class OrcFileFormat
     val capacity = sqlConf.orcVectorizedReaderBatchSize
 
     OrcConf.IS_SCHEMA_EVOLUTION_CASE_SENSITIVE.setBoolean(hadoopConf, sqlConf.caseSensitiveAnalysis)
+    OrcConf.FORCE_POSITIONAL_EVOLUTION.setBoolean(hadoopConf,
+      hadoopConf.getBoolean("spark.hadoop." + OrcConf.FORCE_POSITIONAL_EVOLUTION.getAttribute,
+        false))
```

Comment:
At the last commit, this is added. But, I think this override the user's configuration silently. Is this correct?